purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/DefaultIAPHelperProvider.kt

```diff
@@ -1,13 +1,24 @@
 package com.revenuecat.purchases.galaxy
 
+import com.revenuecat.purchases.common.LogIntent
+import com.revenuecat.purchases.common.log
 import com.revenuecat.purchases.galaxy.utils.GalaxySerialOperation
+import com.samsung.android.sdk.iap.lib.constants.HelperDefine
 import com.samsung.android.sdk.iap.lib.helper.IapHelper
 import com.samsung.android.sdk.iap.lib.listener.OnGetProductsDetailsListener
+import com.samsung.android.sdk.iap.lib.listener.OnPaymentListener
 
 internal class DefaultIAPHelperProvider(
     val iapHelper: IapHelper,
 ) : IAPHelperProvider {
 
+    override fun setOperationMode(
```

Comment:
This function doesn't need to be `@GalaxySerialOperation`. Since the operation doesn't make a network request, Samsung doesn't take this function into account when restricting which operations can be performed in parallel.